backend/test/core/usecase/AddMemberToGroupUseCase.test.ts

```diff
@@ -0,0 +1,81 @@
+import { describe, test, expect, beforeEach, vi } from "vitest";
+import type { GroupRepository } from "src/core/repository/GroupRepository";
+import { GroupRepositoryMock } from "./test-helpers";
+import { AddMemberToGroupUseCase } from "src/core/usecase/AddMemberToGroupUseCase";
+import { GroupMember } from "src/core/entity/GroupMember";
+import { Group } from "src/core/entity/Group";
+
+describe("Add a member to a group (usecase)", () => {
+  let groupRepo: GroupRepository;
+  let testMember: GroupMember;
+
+  beforeEach(() => {
+    groupRepo = new GroupRepositoryMock();
+    testMember = new GroupMember(0, "Pierre");
+  });
+
+  describe("Happy path", () => {
+    test("Given a existing group id, when adding a valid username, it should return the updated group", async () => {
+      // Arrange
+      const member = new GroupMember(1, "John");
+      const group = new Group(2, "groupName", [member], []);
+      groupRepo.findById = vi.fn().mockResolvedValue(group);
+      groupRepo.save = vi.fn(async (group) => Promise.resolve(group));
+      groupRepo.addMember = vi.fn(async (memberName, group) => {
+        return Promise.resolve(
+          new Group(2, "groupName", [member, testMember], [])
+        );
+      });
```

Comment:
```suggestion
      groupRepo.addMember = vi.fn().mockResolvedValue(new Group(2, "groupName", [member, testMember], [])
);
```

Since the arguments aren't used, you can use this instead